app/json_schema/example_upload.json

```diff
@@ -2,7 +2,7 @@
   "author": "author",
   "project": "project",
   "optimization_id": "12345",
-  "type": "optimization_start",
```

Comment:
This means request_type, the state is given by the server, please undo this change